R/translate.R

```diff
@@ -60,10 +60,19 @@ deepl_translate <- function(path,
     formality,
     values = c("default", "more", "less", "prefer_more", "prefer_less")
   )
-
-  # protect Hugo shortcodes ----
   temp_markdown_file <- withr::local_tempfile()
   markdown_lines <- brio::read_lines(path)
+  temp_markdown_file <- withr::local_tempfile()
+
+  # change on char equation to something longer and
```

Comment:
why? shouldn't the math protection make this useless?

Reply:
ok, I'll revert to previous state,  you're right it is not useful